Misc/NEWS.d/next/Core and Builtins/2023-06-02-15-15-41.gh-issue-104812.dfZiG5.rst

```diff
@@ -0,0 +1,8 @@
+The "pending call" machinery now works for all interpreters, not just the
+main interpreter, and runs in all threads, not just the main thread. Some
+calls are still only done in the main thread, ergo in the main interpreter.
+This change does not affect the existing public C-API
+(``Py_AddPendingCall()``) which still only targets the main thread. The new
+functionality is meant strictly for internal use.  This change brings the
```

Comment:
Pending calls can run anything and cause state changes outside the normal flow of execution.  Consequently, it needs to be used carefully because it isn't clear what the potential risks are.  So until we take the time to figure all that out, we are keeping it strictly internal-only.

Reply:
I've clarified the note.